Lib/asyncio/base_futures.py

```diff
@@ -41,6 +42,9 @@ def format_cb(callback):
     return f'cb=[{cb}]'
 
 
+_repr_running = set()
```

Comment:
Might be worth adding a brief comment here to clarify the purpose; e.g. "Used for guarding against recursive reprs from futures" or something along those lines. The name `_repr_running` isn't overly clear by itself without knowing the context of the internal set in `reprlib.recursive_repr()`.

Reply:
Sure, I'll add. Plus, I want to describe why `reprlib.recursive_repr` decorator is not applicable (mine first approach was just applying it).

Reply:
I would say that adding `_asyncio.Task.__module__` allows us to use `reprlib.recursive_repr` but it is a more invasive change.